sympy/integrals/integrals.py

```diff
@@ -758,9 +763,19 @@ def _eval_integral(self, f, x, meijerg=None, risch=None, manual=None,
             else:
                 if i:
                     # There was a nonelementary integral. Try integrating it.
-                    return result + i.doit(risch=False)
+                    a="ans"
```

Comment:
@ashutoshsaboo I might be wrong, but I think you checked out a new branch from that other branch for which you have already submitted a PR.